src/main/java/move/Paper.java

```diff
@@ -11,9 +11,9 @@ public Paper(String name) {
     }
 
     public String playAgainst(Move opponentMove) {
-        if (opponentMove.getName().equals("rock")) {
+        if (opponentMove.getName().equals(Moves.ROCK.name)) {
```

Comment:
If you wanted you could statically import 'Moves.ROCK' (intellij might help you do this too). Then you can just have `.equals(ROCK.name)`.

Reply:
You don't need to go through the name. You can compare enums directly.
So you could write
`if(opponentMove == Moves.ROCK) {...`

Reply:
@felipesere I tried that but didn't work?

Reply:
@gemcfadyen so I tried this in Paper class:

import static rockPaperScissors.move.Moves.ROCK

but looks like IntelliJ doesn't like it cause it keeps removing itself automatically? Or am I doing wrong?

Reply:
Clarifying my point - If you 'alt' 'enter' over `Moves.ROCK.name`, intellij will give you the option to `Add on demand static option for moves.Moves`. If you say ok, it will insert `import static move.Moves.*;` at the top of your file, then remove the type prefix before ROCK. 
So instead of having Moves.ROCK.name, you will have ROCK.name.

Reply:
What Felipe is saying, is that you don't actually need to compare the name specifically. Because the name is the same as the Enum name (ROCK, PAPER, SCISSORS), you can actually just compare the Enum itself, so you could do `opponentMove === ROCK rather than calling `.name()` on them both.